src/data_operations.py

```
@@ -0,0 +1,100 @@
+import pandas as pd
+import time
+from datetime import datetime
+from io import StringIO
+import pickle
+from fredapi import Fred
```

Ruff (0.8.2)

`fredapi.Fred` imported but unused

Remove unused import: `fredapi.Fred`

(F401)